ha_cluster_exporter.yaml

```diff
@@ -1,12 +1,16 @@
 # sample config
-port: "9664"
-address: "0.0.0.0"
-log-level: "info"
+web:
+  listen-address: "0.0.0.0:9664"
+  telemetry-path: "/metrics"
+  config:
+    file: "/etc/ha_cluster_exporter.web.yaml"
```

Comment:
hmm, where is this file coming from?

Reply:
The format is defined in the README.
If it is not existing the exporter will basically start without it (TLS/basic-auth disabled).